griptape/templates/tasks/toolkit_task/system.j2

```diff
@@ -8,16 +8,14 @@ Actions: <JSON array of actions that MUST follow this schema: {{ actions_schema
 {{ stop_sequence }}: <action outputs>
 "Thought", "Actions", "{{ stop_sequence }}" must always start on a new line.
 
+You must use the following format when providing your final answer:
+Answer: <final answer>
 {% endif %}
 Repeat executing actions as many times as you need.
 If an action's output contains an error, you MUST ALWAYS try to fix the error by executing another action. 
-You must use the following format when providing your final answer:
-Answer: <final answer>
 
 Be truthful. ALWAYS be proactive and NEVER ask the user for more information input. Keep using actions until you have your final answer.
 NEVER make up actions, action names, or action paths. NEVER make up facts. NEVER reference tags in other action input values.
-
-Actions might store their output in memory as artifacts (with `memory_name` and `artifact_namespace`). If action output is stored in memory, ALWAYS try to pass it to another action. NEVER make up memory names or artifact namespaces.
```

Comment:
This was causing the LLM to hallucinate these keys into the Tool schema. The Tool schema is descriptive enough, we don't need to mention them here.